src/de/jost_net/JVerein/gui/action/FormularDeleteAction.java

```diff
@@ -55,6 +57,16 @@ public void handleAction(Object context) throws ApplicationException
         return;
       }
 
+      DBIterator<Spendenbescheinigung> spb = Einstellungen.getDBService()
+          .createList(Spendenbescheinigung.class);
+      spb.addFilter("formular = ?", new Object[] { f.getID() });
+      if (spb.size() > 0)
+      {
+        throw new ApplicationException(String.format(
+            "Forular '%s' kann nicht gel�scht werden. Es ist bei %d Spendenbeschinugung(en) hinterlegt.",
```

Comment:
Da ist noch ein Tippfehler in Spendenbeschinugung(en)

Reply:
Habe ich alle korrigiert.